diff --git a/build.sbt b/build.sbt
index 6c5ea63f9..24a036b1d 100644
--- a/build.sbt
+++ b/build.sbt
@@ -248,6 +248,7 @@ lazy val externalvm = project
     //      and https://github.com/thesamet/sbt-protoc/issues/6
     //      That is why generate protobuf code in another folder
     //      (`protobuf`).
+    Compile / PB.runProtoc := ((args: Seq[String]) => Process("@protobuf@/bin/protoc", args) !),
     Compile / PB.targets := Seq(
       scalapb.gen() -> (Compile / sourceManaged).value / "protobuf"
     )
